apps/studio/src/server/modules/resource/resource.service.ts

```diff
@@ -81,8 +81,9 @@ export const getFullPageById = async (args: {
   }
 
   return getById(args)
-    .where("Resource.mainBlobId", "is not", null)
-    .innerJoin("Blob", "Resource.mainBlobId", "Blob.id")
+    .where("Resource.versionId", "is not", null)
+    .innerJoin("Version", "Resource.versionId", "Version.id")
```

Comment:
issue: also same here

```suggestion
    .where("Resource.publishedVersionId", "is not", null)
    .innerJoin("Version", "Resource.publishedVersionId", "Version.id")
```

Reply:
My bad, didn't push the updated types